source/azure_iot_json_reader.c

```diff
@@ -0,0 +1,237 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_json_reader.c
+ * @brief Implementation of the Azure IoT JSON reader.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "azure_iot_json_reader.h"
+
```

Comment:
Put this include first and the all other.

generally it is prefered to honor the includes of the interface by its implementation, and then use other include.